qiskit/transpiler/passes/routing/lookahead_swap.py

```diff
@@ -88,6 +88,8 @@ def run(self, dag):
             compatible with the DAG
         """
         coupling_map = self.coupling_map
+        search_depth = self.search_depth
+        search_width = self.search_width
```

Comment:
It's not clear to me why we need to locally alias these instance variables inside the run method (it's not like it's a local copy that gets modified for just the lifetime of `run()`). But not worth blocking over because it's the pattern here already it's something easily fixed in a followup if needed